### ISSUE TITLE
`Merge cells` button should be disabled

### ISSUE THREAD

**Issue by panr** [ckeditor/ckeditor5#6677]
## 📝 Provide detailed reproduction steps (if any)

1. Select table headers (`th`) and cells (`td`).
2. Look at the `Merge cells` button in the table toolbar.

### ✔️ Expected result

In this case button should be disabled, because we don't allow to merge those (`th` and `td`).

### ❌ Actual result

Button is enabled, but doesn't do anything...

![image](https://user-images.githubusercontent.com/1303365/80354863-988bb580-8877-11ea-90bf-8ac7af49fa2e.png)

## 📃 Other details

* Browser: Edge (Chromium)
* OS: macOS Mojave

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by panr** [comment]
Somehow related: https://github.com/ckeditor/ckeditor5/issues/6666

**Comment by jodator** [comment]
I'd go with duplicates #6666 (better number) and maybe reword the #6666 issue as I feel that those tickets are about the same.

**Comment by panr** [comment]
OK. I'm closing this issue.
